### PR TITLE
API RFC tweaks

### PR DESCRIPTION
**Pull request by cloveras**
Trying to make it match the API RFC: https://github.com/vippsas/vipps-rfc/blob/master/0001-api-guidelines.md

I'd also like to rename the repo to "vipps-order-management-api".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
 
-The Order Management API allows merchants to send rich receipt information to existing Vipps transaction. This information is shown to the customer in the app in their order history.
-This functionality is available for both recurring and direct payments, but not for passthrough payments.
+The Vipps Order Management API allows merchants to send rich receipt information to existing Vipps transactions. This information is shown to the customer in the app in their order history.
+This functionality is available for both
+[recurring](https://github.com/vippsas/vipps-ecom-api)
+and
+[direct payments](https://github.com/vippsas/vipps-recurring-api),
+but not for
+[passthrough payments](https://github.com/vippsas/vipps-psp-api).
```
Comment (tht13) [inline README.md:11]: passthrough isn't allowed to use this

**Review by tht13 — COMMENTED** [review]
please don't rename the swagger file, we have a reference to this for building the api in the backend

File: README.md (hunk 1 above)
Comment (cloveras) [reply README.md:11]: Renaming: Ok, but can the swagger file be placed in the "docs" directory, so Swagger UI and ReDoc works?

I see I copied and pasted wrong, will fix.

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        }
+      }
+    }
+  }
+}
+
+window.onload = function() {
+  // Build a system,
+  const ui = SwaggerUIBundle({
+    url: 'https://vippsas.github.io/vipps-ecom-api/swagger.yaml',
```
Comment (tht13) [inline docs/index.html:51]: @cloveras this is the url that needs changing for swagger, so moving it to docs won't work unless you change this too

you can also reference the raw file without putting it in docs via

`https://raw.githubusercontent.com/vippsas/order-management-api/main/reference/order-management-api.v1.yaml`

**Comment by cloveras** [comment]
Outdated.

**Pull request closed without merging**